packages/types/Addressbook.d.ts

```diff
@@ -0,0 +1,41 @@
+interface AddressbookEntry {
+    id: string,
+    label: string,
+    description: string,
+    operation?: string,
+    user?: string,
+    user_details?: object,
+}
+
+interface Channel {
+    id: string,
+    entry: string,
+    label: string,
+    type: ChannelType,
+    priority: number,
+    min_importance: number,
+    details: ChannelDetail,
+    timeout: number,
+}
+
+interface ChannelDetail {
+    info?: string,
+    email?: string,
+    forward_to_group?: string,
+    forward_to_user?: string,
+    phone?: string,
+}
+
+declare enum ChannelType{
```

Comment:
We should think about either making the constants into enums or putting the channeltypes into the constants module